data-plane/core/pom.xml

```diff
@@ -102,20 +98,43 @@
         </exclusion>
       </exclusions>
     </dependency>
+
     <dependency>
-      <groupId>io.cloudevents</groupId>
-      <artifactId>cloudevents-http-vertx</artifactId>
+      <groupId>io.smallrye.reactive</groupId>
+      <artifactId>smallrye-mutiny-vertx-web-client</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.smallrye.reactive</groupId>
+      <artifactId>smallrye-mutiny-vertx-micrometer-metrics</artifactId>
     </dependency>
```

Comment:
note: this is not present in https://maven.repository.redhat.com/ga/io/smallrye/reactive/

Reply:
Hi @cescoffier, will `smallrye-mutiny-vertx-micrometer-metrics` be productized?

Reply:
No.

Reply:
Thanks, what's the recommended migration path?